keanu-project/src/test/java/io/improbable/keanu/vertices/intgrtensor/probabilistic/PoissonVertexTest.java

```diff
@@ -0,0 +1,57 @@
+package io.improbable.keanu.vertices.intgrtensor.probabilistic;
+
+import io.improbable.keanu.vertices.dbltensor.KeanuRandom;
+import org.apache.commons.math3.stat.descriptive.SummaryStatistics;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class PoissonVertexTest {
+    private final Logger log = LoggerFactory.getLogger(io.improbable.keanu.vertices.intgr.probabilistic.PoissonVertexTest.class);
```

Comment:
nit: should be `intgrtensor`

Reply:
That was intellij trying to help us out. Good work!